ios/Gekidou/Sources/Gekidou/Networking/ShareExtension+Post.swift

```diff
@@ -56,6 +61,7 @@ extension ShareExtension {
                                 filename,
                                 id
                             )
+                            return "There was an error when trying to upload. Please try again"
```

Comment:
before the other two returns you added a `notifyFailureNow` call, but not here, why?